### PR TITLE
Database ms

### PR DESCRIPTION
**Pull request by RicardoUsbeck**

### CODE REVIEW

File: database-ms/pom.xml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+			</exclusion> </exclusions> </dependency> -->
+
+<!-- 		<dependency> -->
+<!-- 			<groupId>org.springframework.boot</groupId> -->
+<!-- 			<artifactId>spring-boot-starter-log4j2</artifactId> -->
+<!-- 		</dependency> -->
+
+		<!-- <dependency> <groupId>junit</groupId> <artifactId>junit</artifactId> 
+			<scope>test</scope> </dependency> -->
+
```
Comment (RicardoUsbeck) [inline database-ms/pom.xml]: @sepidetari please delete unused code

File: database-ms/src/main/java/org/dice_research/sask/database_ms/DbMSApplication.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
+package org.dice_research.sask.database_ms;
```
Comment (RicardoUsbeck) [inline database-ms/src/main/java/org/dice_research/sask/database_ms/DbMSApplication.java]: This class does not seem to be formatted according to the rules

File: database-ms/src/main/java/org/dice_research/sask/database_ms/RDFTriples/TripleDTO.java [hunk 3]
```diff
@@ -0,0 +1,22 @@
+/**
```
Comment (RicardoUsbeck) [inline database-ms/src/main/java/org/dice_research/sask/database_ms/RDFTriples/TripleDTO.java]: What does TDO stand for?

**Comment by RicardoUsbeck** [comment]
I do not understand the purpose.

File: database-ms/src/main/java/org/dice_research/sask/database_ms/DbMSApplication.java (hunk 2 above)
Comment (sepidetari) [reply database-ms/src/main/java/org/dice_research/sask/database_ms/DbMSApplication.java]: The structure of the this class is similar to the other micro services application class.

File: database-ms/src/main/java/org/dice_research/sask/database_ms/RDFTriples/TripleDTO.java (hunk 3 above)
Comment (sepidetari) [reply database-ms/src/main/java/org/dice_research/sask/database_ms/RDFTriples/TripleDTO.java]: DTO stands for Data transfer object. I wanted to keep the structure the same as other ms so I named it tripleDTO.

**Comment by sepidetari** [comment]
Thank you for your review. I will add test unit and delete unused code. 
About "I do not understand the purpose" I am not sure if I completely understand what you mean by purpose. The query method is aimed to check if the triples are stored in the existing graph or not that is why it returns the whole graph to check if it added to the existing graph or not. It will be updated according to the need in future.

**Comment by AndreSonntag** [comment]
Actually the pull request should be close. We dicussed this a couple of weeks ago. I don't know why the pull request is still open. See #23 @sepidetari @Suganya31

**Comment by RicardoUsbeck** [comment]
So your and Kevin's review is positive?

**Comment by AndreSonntag** [comment]
Not negative or positive.

Main cause for this is that we merged the database branch into the repository branch for some weeks ago. This helped us to test the functionality of the database. Since this rejected pull request, we did some 
new changes to this microservice. In my personal view is a further review or this pull request in general 
unnecessary. The database microservice will be included in the next pull request of the repository branch.

**Comment by RicardoUsbeck** [comment]
So we delete this pull request?

**Comment by guru5590** [comment]
Hi Andre,

Are you planning to merge repository branch to master this week?


Thanks,
Guru

**Pull request closed without merging**